cli_tools/common/utils/param/network_resolver_test.go

```diff
@@ -0,0 +1,148 @@
+//  Copyright 2021 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package param
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/api/compute/v1"
+
+	"github.com/GoogleCloudPlatform/compute-image-tools/cli_tools/mocks"
+)
+
+func TestNetworkResolver_PassingCases(t *testing.T) {
+	type networkParams struct{ project, name string }
+	type subnetParams struct{ project, region, name string }
```

Comment:
These type names are easily confused with CLI params. Something like getNetworkComputeParams would be clearer.

Reply:
updated